.github/workflows/format.yml

```diff
@@ -26,4 +26,4 @@ jobs:
       - name: Install dependencies
         run: yarn install
       - name: Run check
-        run: yarn format:check
+        run: cd ./packages/pancake-uikit && yarn format:check
```

Comment:
I think in this case we need to standardize the call at top-level. Lets change the top level command from `"format": "lerna run format",` to `"format:check": "lerna run format:check",`